### ISSUE TITLE
AutoCompletion of Attribute-types includes `Attribute` suffix when not needed/required

### ISSUE THREAD

**Issue by josteink** [razzmatazz/csharp-language-server#7]
Consider the following code:

````c#
[Route("Demo")]
public class DemoController : Contrller
{
    [HttpGet]
    [Router("DemoOperation")]
    public IActionResult DemoOperation()
    {
        // some logic
    }
}
````

Now try chaning the `[HttpGet]` attribute into a `[HttpPost]` attribute, by deleting `Get`, entering `P` and wait for auto-completion to kick in.

**Observed that:**

* `HttpPost` is **not** suggested
* `HttpPostAttribute` **is** suggested
* It is *harder* to write `[HttpPost]` than without auto-completion. Oops?

Basically, unless you are willing to live with the needlessly verbose attribute-name, auto-completion now kinda works against you.

Since attribute-completion is somewhat context-sensitive, I'm not sure if there are any simple solutions here... But if there is, that would really be appreciated :smile:

**Comment by razzmatazz** [comment]
Yeah.. I will need to check how omnisharp does that and will steal the code :)

**Comment by razzmatazz** [comment]
does it do the same for [Flags] ? because I tried adding a [Flags] on enum and it worked..

probably your issue is with classes, right?

**Comment by josteink** [comment]
Yeah. I’ve tested with basic classes and not with enum-flags yet.

**Comment by josteink** [comment]
Old issue is old. Not sure if it's still even an issue. Abandoning.
